plugins/org.python.pydev.core/src/org/python/pydev/core/docutils/PySelection.java

```diff
@@ -547,6 +547,9 @@ public String getLineContentsToCursor(boolean removeComments, boolean removeLite
                 StringTokenizer tokenizer = new StringTokenizer(insideParentesisTok, ",");
                 while (tokenizer.hasMoreTokens()) {
                     String tok = tokenizer.nextToken();
+                    if (tok.contains("]")) {
```

Comment:
I'm not sure this is the best approach... 

Can you create a unit-test (in `org.python.pydev.editor.actions.PySelectionTest`) which checks for some more related cases on what is happening in `org.python.pydev.core.docutils.PySelection.getInsideParentesisToks(boolean, int, boolean)` when there are type hints?

-- I believe that in general we want to drop anything related to type hints and I'm not sure the current fix covers all the cases needed...